pkg/controller/dataimportcron-controller_test.go

```diff
@@ -310,6 +310,9 @@ var _ = Describe("All DataImportCron Tests", func() {
 
 			verifyCronJobContainerImage("old-image")
 			verifyCronJobContainerImage("new-image")
+
```

Comment:
why test cronjob if it shouldn't be created? split it to another test anyway

Reply:
fixed, wrote another test